src/middleware/jwtValidator.js

```diff
@@ -0,0 +1,33 @@
+let jwt = require('jsonwebtoken');
+const config = require('../database/config/config.js');
+
+let checkToken = (req, res, next) => {
+  let token = req.headers['x-access-token'] || req.headers['authorization']; // Express headers are auto converted to lowercase
+  if (token.startsWith('Bearer ')) {
+    // Remove Bearer from string
+    token = token.slice(7, token.length);
+  }
+
+  if (token) {
+    jwt.verify(token, config.development.password, (err, decoded) => {
```

Comment:
This line is funny. We have a `JWT_KEY` defined in our `.env` file. Use that instead of `config.development.password`